### PR TITLE
Fix test discovery

### PR DESCRIPTION
**Pull request by vijayphoenix**
Since we are adding `.vscode` folder to `datasets/tensorflow_datasets` instead of `datasets`, the following changes are needed to fix test discovery.

### CODE REVIEW

File: tensorflow_datasets/.vscode/settings.json [hunk 1]
```diff
@@ -19,11 +19,10 @@
...
-        "tensorflow_datasets",
         "-vv",
         "--continue-on-collection-errors",
-        "--ignore=tensorflow_datasets/testing/test_utils.py",
-        "--ignore=tensorflow_datasets/scripts/documentation/build_api_docs_test.py"
+        "--ignore=testing/test_utils.py",
+        "--ignore=scripts/documentation/build_api_docs_test.py"
     ],
     "files.watcherExclude": {
         "**/.git/**": true,
```
Comment (vijayphoenix) [inline tensorflow_datasets/.vscode/settings.json:28]: As `.git/` and `docs/` files are in the `datasets` folder, so maybe we can remove here.

**Comment by Conchylicultor** [comment]
Thank you for catching this. I mistakenly moved the `.vscode/` in the wrong directory.

@vijayphoenix, As you have more experience working with VSCode, do you think it would be better to have `.vscode/`, `.pylintrc`,... in `datasets/.vscode` or to keep it in `datasets/tensorflow_datasets/` ?

I feel having it at the top level might be more discoverable ?

**Comment by vijayphoenix** [comment]
Yes, it would be better to move all the additional files in the `datasets` folder.

**Comment by vijayphoenix** [comment]
Fixed by https://github.com/tensorflow/datasets/commit/3e3fc7ccf8e41e1b00ed5be56cf51f96908b31a7

**Pull request closed without merging**